apps/web/src/app/[locale]/page.tsx

```
@@ -1,5 +1,192 @@
+	HeroSubtitle,
+	HeroTitle,
+} from '../components/hero'
+import { Highlights } from '../components/highlights'
+import { Link } from '../components/link'
+import { BlogOverview } from './blog/blocks/blog-overview'
+import { Testimonials } from './testimonials/blocks/testimonials'
+import { WorkOverview } from './work/blocks/work-overview'
 
-export { generateMetadata }
+export async function generateMetadata(props: {
+	params: Promise<WithLocale<{}>>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)